### ISSUE TITLE
Get API docs built to markdown

### ISSUE THREAD

**Issue by crazy4pi314** [qsharp-community/qram#27]
Check to see how/what tool we can use to  convert API docs to markdown files.

**Comment by amirebrahimi** [comment]
Have you considered [docfx](https://github.com/dotnet/docfx)?

**Comment by crazy4pi314** [comment]
Yeah thats what I want to use basically, but we need a way to scrape the API comments from the source. I think either myself or @cgranade might have a complier extension floating around that does that. Just need to get it documented 😄

**Comment by cgranade** [comment]
I had a pretty rough experiment at https://github.com/microsoft/qsharp-runtime/pull/253, but haven't had the chance yet to move it to qsharp-compiler as suggested, or to address the feedback so far. The biggest is that the tool picks up everything from assembly references as well as the current compilation unit; as per @bettinaheim's feedback, that can be fixed by looking at the Source property to check for `.EndsWith(".qs")` versus `.EndsWith(".dll")`.

**Comment by crazy4pi314** [comment]
Items left after #49
- [ ] Configure DocFX or similar to consume the Q# generated markdown
- [ ] Get GH actions setup to build the docs and serve them as a GH pages site
